components/hab/src/cli/hab.rs

```diff
@@ -1,42 +1,154 @@
-mod bldr;
-mod cli;
-mod config;
-mod file;
+pub mod bldr;
+pub mod cli;
+pub mod config;
+pub mod file;
 pub mod license;
 pub mod origin;
 pub mod pkg;
-mod plan;
-mod ring;
+pub mod plan;
+pub mod ring;
 pub mod studio;
 pub mod sup;
 pub mod svc;
 #[cfg(test)]
 mod tests;
-mod user;
+pub mod user;
 pub mod util;
 
-use self::{bldr::{Bldr,
-                  ConfigOptBldr},
-           cli::{Cli,
-                 ConfigOptCli},
-           config::{ConfigOptServiceConfig,
-                    ConfigOptServiceConfigApply,
-                    ServiceConfig,
-                    ServiceConfigApply},
-           file::{ConfigOptFile,
-                  File},
+use self::{bldr::{ChannelCreate,
+                  ChannelDemote,
+                  ChannelDestroy,
+                  ChannelList,
+                  ChannelPromote,
+                  ConfigOptChannelCreate,
+                  ConfigOptChannelDemote,
+                  ConfigOptChannelDestroy,
+                  ConfigOptChannelList,
+                  ConfigOptChannelPromote,
+                  ConfigOptJobCancel,
+                  ConfigOptJobDemote,
+                  ConfigOptJobPromote,
+                  ConfigOptJobStart,
+                  ConfigOptJobStatus,
+                  JobCancel,
+                  JobDemote,
+                  JobPromote,
+                  JobStart,
+                  JobStatus},
```

Comment:
While I generally dislike `*` imports as a rule, I think it could be helpful for the `use` statements that we've got in this corner of the code, simply because of the sheer volume of imports, as well as the fact that many of those imports are auto-generated `ConfigOpt` ones, which tends to make things look a bit confusing.

So, this highlighted code could be collapsed to `use self::bldr::*`, with similar replacements for the other module declarations, both in this file and in others that this PR is touching.

What do you think?

Reply:
Agree in this case!